src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -282,9 +284,15 @@ export class HuiGlanceCard extends LitElement implements LovelaceCard {
 
     const name = entityConf.name ?? computeStateName(stateObj);
 
+    const hasAnyAction =
+      !entityConf.tap_action ||
+      hasAction(entityConf.tap_action) ||
+      hasAction(entityConf.hold_action) ||
```

Comment:
There is also a default hold action it seems?

Reply:
I believe this code is correct as-is. 

Glance does create a default hold action for each entity, but it creates it in the entityConf, so this code does detect the default hold action. 

If user don't specify any actions in the config, then here entityConf will have `hold_action` only, and hasAction detects that it has a hold action. 

If user specifies `hold_action: none`, than the only action left is the tap action, which is the weird case since `hasAction(entityConf.tap_action)` will return `false`, but actionHandler still handles a tap action even when it is undefined (which is why I add the extra case to check for this implied tap action).

Reply:
Right, you are correct! Sorry :-)